data/json/items/armor/bandolier.json

```diff
@@ -1,24 +1,112 @@
 [
   {
-    "id": "shotbelt",
+    "id": "bandolier_pistol",
+    "type": "ARMOR",
+    "name": "pistol bandolier",
+    "description" : "An embroidered leather bandolier for keeping revolver cartridges close to hand.  If this doesn't make you feel like a cowboy nothing will.",
+    "weight": 140,
+    "volume": 1,
+    "price": 1900,
+    "material": "leather",
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "TORSO" ],
+    "coverage": 10,
+    "encumbrance": 6,
+    "material_thickness": 1,
+    "use_action": {
+        "type": "bandolier",
+        "capacity": 18,
+        "ammo": [ "32", "38", "44" ]
```

Comment:
This should probably include other rounds used by revolvers like 9mm, .45, and 8x40mm.